### ISSUE TITLE
Add back the "short" show output

### ISSUE THREAD

**Issue by piotrb** [hashicorp/terraform#23454]
### Current Terraform Version
```
Terraform v0.12.16
```

### Use-cases

With the previous "short" output from `show` I was able to at a glance see what resources are being updated and how. This was extremely helpful. This way its easy to spot changes in resources that are unexpected by the recent changes to the TF files.

### Attempted Solutions

N/A this was a removed feature.

### Proposal

If you guys are that keen on keeping the "long" output, I'd like to see a new command line flag .. eg `--short` or something that would allow seeing the old output.

### References

Original change that removed this behavior - #23292

**Comment by lifeofguenter** [comment]
One possible workaround (for now):

```bash
terraform show -no-color | grep -E '^\s*# '
```

Even though I enjoyed the previous output format, the `-` and `~` were sometimes not clear enough on smaller fonts.

**Comment by nhudacin** [comment]
I agree with this! We leveraged the shorter version to roll up a summary report on a PR.
